src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.tsx

```diff
@@ -3,15 +3,16 @@ import { FormState } from "../../../constants/FormState";
 import { AssetManagementActions } from "../AssetManagementActions";
 import { ActionSelectionForm } from "./FormVariants/ActionSelectionForm";
 import { EndorseBeneficiaryForm } from "./FormVariants/EndorseBeneficiary";
+import { EndorseTransferForm } from "./FormVariants/EndorseTransferForm";
+import { NominateBeneficiaryHolderForm } from "./FormVariants/NominateBeneficiaryHolder";
 import { SurrenderForm } from "./FormVariants/SurrenderForm";
 import { TransferHolderForm } from "./FormVariants/TransferHolderForm";
-import { NominateBeneficiaryHolderForm } from "./FormVariants/NominateBeneficiaryHolder";
 
 interface AssetManagementFormProps {
   beneficiary?: string;
   holder?: string;
-  approvedBeneficiary?: string;
-  approvedHolder?: string;
+  approvedBeneficiary: string;
+  approvedHolder: string;
```

Comment:
probably need to change back when no fallback of empty string

Reply:
changed back as have moved the fallback to where we call the function